src/google/protobuf/arenastring.cc

```diff
@@ -264,14 +264,12 @@ void ArenaStringPtr::ClearToDefault(const LazyString& default_value,
 }
 
 const char* EpsCopyInputStream::ReadArenaString(const char* ptr,
+                                                int size,
```

Comment:
Is this change required?

Reply:
yes the fast parse loop already reads the size.